xsd_glue/domain_getinfo_plugin_v1/domain_getinfo_stubs_v1.c

```diff
@@ -71,6 +71,7 @@ xsd_glue_failwith (xc_interface * xch, const char *func, unsigned int line)
         free (str);
 
         caml_raise_with_arg (*caml_named_value ("xsg.error_v1"), msg);
+        CAMLreturn0;
```

Comment:
noreturn seems to have been correct here, it does always raise, so now you might get a compiler warning about unreachable code instead.

Try `CAMLnoreturn;`